### PR TITLE
feat(content): Add new civilian hails, revise others, and add some more disabled hails for pirates

### PR DESCRIPTION
**Pull request by bene-dictator**
**Content (Artwork / Missions / Jobs)**
This PR modified human hails.

## Summary
* Adds three new civilian hails (two on zero-g cricket and one on the Quarg, derived from a previous hail).
* Adds one new variation on a Navy hail.
* Removes one hail but integrates its text into another hail set and adds some more to them (as those "in Soviet Russia"-type jokes were happening too often).
* Revises some other hails & cuts down on some truly absurd variation I put in a couple years ago into the navy hails.
* Adds some new variation to a bunch of other hails.
* Finally, adds new sets of pirate disabled hails and makes the pirate governments use those hails.

## Testing Done
Tested some through the James on Swizzle Six. I will test others more thoroughly when I get the time.

### CODE REVIEW

File: data/human/hails.txt [hunk 1]
```diff
@@ -6098,6 +6130,25 @@ phrase "hostile pirate"
...
 		"!"
 
+phrase "hostile disabled pirate"
+	word
+		"Why isn't the ship moving?!"
+		"Move, you goddamned ship!"
+		"Why aren't the guns firing?!"
+		"Come on, fire the bloody guns!"
+		"Fire! Fire! Fire the damn guns! Why aren't you firing?"
+		"Why aren't the guns firing?"
```
Comment (Hurleveur) [inline data/human/hails.txt:6140]: You used this twice

File: data/human/hails.txt [hunk 2]
```diff
@@ -1982,24 +1973,120 @@ phrase "friendly civilian"
...
+		"even the most powerful warship won't save you from a knife in the back"
+		"nine out of ten pirates die within their first year"
+		"Republic Intelligence has their eyes everywhere"
+		"the law will always catch up to those who try to bypass it"
+	word
+		"."
+
+phrase "friendly civilian"
+	word
+		"Those Quargs are pretty strict with their technology. They"
```
Comment (tibetiroka) [inline data/human/hails.txt:2000]:
```suggestion
		"Those Quarg are pretty strict with their technology. They"
```

File: data/human/hails.txt [hunk 3]
```diff
@@ -1982,24 +1973,120 @@ phrase "friendly civilian"
...
+	word
+		" make it pretty clear that they "
+		" monitor their outfits meticulously, and they "
+		" don't want their outfits to proliferate, and they "
+		" restrict their outfits to their own ships, and "
+		" " 4
+	word
+		"will arrest you if they catch a whiff of any of their equipment on your ship"
+		"imprison anyone they catch carrying their outfits"
+		"will put you behind bars forever if you've stolen their outfits"
```
Comment (tibetiroka) [inline data/human/hails.txt:2014]: Maybe you could use some word other than outfit?

File: data/human/hails.txt [hunk 4]
```diff
@@ -6098,6 +6130,27 @@ phrase "hostile pirate"
 	word
 		"!"
 
+phrase "hostile disabled pirate"
```
Comment (tibetiroka) [inline data/human/hails.txt:6133]:
```suggestion


phrase "hostile disabled pirate"
```

**Review by tibetiroka — CHANGES_REQUESTED** [review]

File: data/human/hails.txt [hunk 5]
```diff
@@ -1982,24 +1973,120 @@ phrase "friendly civilian"
...
+	word
+		" and the way the crowd roars as it happens"
+		", the crowd going wild"
+		", and the release you feel as you see the ball getting knocked right out of the sphere"
+		", how each test is full of tension and fortunes that are decided by each bowl and hit"
+		"" 6
+	word
+		"; there's nothing like it in the galaxy."
+		"; I don't get the hate for it."
+		"... There are some truly phenomenal players out there."
```
Comment (roadrunner56) [inline data/human/hails.txt:2086]:
```suggestion
		"...There are some truly phenomenal players out there."
```

File: data/human/hails.txt [hunk 6]
```diff
@@ -1982,24 +1973,120 @@ phrase "friendly civilian"
...
+		"... it's an amazing sport."
```
Comment (roadrunner56) [inline data/human/hails.txt:2087]:
```suggestion
		"...it's an amazing sport."
```

File: data/human/hails.txt [hunk 7]
```diff
@@ -1982,24 +1973,120 @@ phrase "friendly civilian"
...
+		"... it's like chess played in space."
```
Comment (roadrunner56) [inline data/human/hails.txt:2088]:
```suggestion
		"...it's like chess played in space."
```

**Review by roadrunner56 — CHANGES_REQUESTED** [review]

File: data/human/hails.txt (hunk 5 above)
Comment (bene-dictator) [reply data/human/hails.txt:2086]: I don't usually see an elipsis without a space after it, but apparently this is a thing.

File: data/human/hails.txt [hunk 8]
```diff
@@ -2089,6 +2091,40 @@ phrase "friendly civilian"
...
+	word
+		"If you ever get the chance, you should"
+		"One thing I'd like to do before I kick the bucket is to"
+		"Before you die, you've got to"
+		"For my next trip, I'm going to"
+	word
+		" "
+	word
+		"see the Quarg ringworld in Enif. Isn't it amazing what they can do and we can't?"
+		"see the Quarg ringworld down south. It's such a magnificent feat of engineering."
```
Comment (tibetiroka) [inline data/human/hails.txt:2104]: The period shouldn't be there and the question mark will need a special replacement.

File: data/human/hails.txt [hunk 9]
```diff
@@ -2089,6 +2091,40 @@ phrase "friendly civilian"
...
+		"do a jungle trek through Amazon"
+		"tour the continents of Pearl and experience the biospheres of a hundred different planets"
+		"meet some of the Quarg around Tarazed"
+		"visit Luna and its enormous shipyard"
+		"see where the Niel Armstrong, one of the first men to leave Earth, stepped foot on Luna"
+		"see the Plot Memorial on Chiron and take a look at the first hyperdrive ever made"
+		"watch the hovercraft races on Sunracer"
+		"relax at the hot springs on Calda"
+		"take a boat up the fjords of Flood"
+		"one of the pirate worlds, like Greenrock or Freedom, and experience life the way anarchists do it"
```
Comment (tibetiroka) [inline data/human/hails.txt:2124]:
```suggestion
		"visit one of the pirate worlds, like Greenrock or Freedom, and experience life the way anarchists do it"
```

**Review by tibetiroka — CHANGES_REQUESTED** [review]

File: data/human/hails.txt (hunk 5 above)
Comment (LazerLit) [reply data/human/hails.txt:2089]: Technically this isn't an ellipsis, it's 'suspension points'. An ellipsis indicates that something is missing and suspension points show hesitation or a pause.

In this specific case I would keep the space but drop the capitalisation, as the sentence is running on from whichever previous phrase was displayed.

```suggestion
		"... there are some truly phenomenal players out there."
```

File: data/human/hails.txt [hunk 10]
```diff
@@ -1982,24 +1974,156 @@ phrase "friendly civilian"
...
+	word
+		"."
```
Comment (bene-dictator) [inline data/human/hails.txt:2126]:
```suggestion
		"."
	replace
		"?." "?"
```

File: data/human/hails.txt [hunk 11]
```diff
@@ -1982,24 +1974,156 @@ phrase "friendly civilian"
...
+	word
+		"If you ever get the chance, you should"
+		"One thing I'd like to do before I kick the bucket is to"
+		"Before you die, you've got to"
+		"For my next trip, I'm going to"
+	word
+		" "
+	word
+		"see the Quarg ringworld in Enif. Isn't it amazing what they can do and we can't?"
+		"see the Quarg ringworld down south. It's such a magnificent feat of engineering."
```
Comment (bene-dictator) [inline data/human/hails.txt:2104]:
```suggestion
		"see the Quarg ringworld down south. It's such a magnificent feat of engineering"
```

File: data/human/hails.txt [hunk 12]
```diff
@@ -1982,24 +1974,158 @@ phrase "friendly civilian"
...
 		"I've heard"
-		"I heard a rumor"
 		"You should know"
 		"They say"
 		"It's said"
+		"I know from experience"
+		"Once you've been on the starlanes long enough you know"
 	word
 		" that "
+		", "
```
Comment (roadrunner56) [inline data/human/hails.txt:1989]: This comma feels kind of weird and unnatural. An example sentence: "Once you've been on the starlanes long enough you know, smuggling is a sure way to find yourself behind bars."

File: data/human/hails.txt [hunk 13]
```diff
@@ -1982,24 +1974,158 @@ phrase "friendly civilian"
...
+		"see the Quarg ringworld down south. It's such a magnificent feat of engineering"
+		"see the dragons of Skymoot"
+		"ski the slopes on Vail"
+		"go to Vail and do some skiing"
+		"go to Vail for a hoverboard course"
+		"go to New Switzerland. The slopes are much better there than Vail"
+		"go skiing on Antipode. It's not as busy as those overpriced resorts in Vail, not to mention cheaper"
+		"do some extreme rock climbing on Maelstrom"
+		"see the ancient cities of old Earth"
+		"tour through the megacities on Earth"
```
Comment (roadrunner56) [inline data/human/hails.txt:2113]:
```suggestion
		"tour through the megacities of Earth"
```

File: data/human/hails.txt [hunk 14]
```diff
@@ -1982,24 +1974,158 @@ phrase "friendly civilian"
...
+		"see where our ancestors came from on old Earth"
+		"do a jungle trek through Amazon"
+		"tour the continents of Pearl and experience the biospheres of a hundred different planets"
+		"meet some of the Quarg around Tarazed"
+		"visit Luna and its enormous shipyard"
+		"see where the Niel Armstrong, one of the first men to leave Earth, stepped foot on Luna"
```
Comment (roadrunner56) [inline data/human/hails.txt:2119]:
```suggestion
		"see where Neil Armstrong, one of the first men to leave Earth, stepped foot on Luna"
```

**Review by roadrunner56 — CHANGES_REQUESTED** [review]
I'm not big on my previous suggested changes, so you can ignore that if you want.

File: data/human/hails.txt [hunk 15]
```diff
@@ -2938,11 +3015,7 @@ phrase "friendly navy"
...
 		"inspections"
 	word
-		" on "
-	word
-		"civilian"
-		"merchant"
-		"private"
+		" on all "
 	word
 		" "
```
Comment (bene-dictator) [inline data/human/hails.txt:3020]:
```suggestion
```

File: data/human/hails.txt (hunk 12 above)
Comment (bene-dictator) [inline data/human/hails.txt:1989]:
```suggestion
```

Comment (bene-dictator) [reply data/human/hails.txt:1989]: I'll need to look at this hail again.

**Review by roadrunner56 — APPROVED** [review]

**Review by tibetiroka — APPROVED** [review]

File: data/human/hails.txt [hunk 16]
```diff
@@ -1982,24 +1974,157 @@ phrase "friendly civilian"
...
+	word
+		"will arrest you if they catch a whiff of any of their equipment on your ship"
+		"imprison anyone they catch with their outfits"
+		"will put you behind bars forever if you've got one of their ships or outfits without their permission"
+		"won't ever let you step foot on a spaceship again if you've got one of their own ships"
+		"don't take any chances when it comes to a misplaced ship or outfit of theirs"
+	word
+		"."
+	replace
+		"It's said that the Quarg make it pretty clear that they" "It's said that the Quarg"
```
Comment (warp-core) [inline data/human/hails.txt:2021]: Why do we not want:
"It's said that the Quarg make it pretty clear that they"
But:
"The Quarg make it pretty clear that they"
"They make it pretty clear that they"
"I'd love to get my hands on some Quarg tech, but they make it pretty clear that they"
Are fine?

Also, the thing with replacing `?!` with `?`; why not just put an `!` at the end of the lines where you want that?

Comment (bene-dictator) [reply data/human/hails.txt:2021]: `It's said` suggests a rumour, `make it pretty clear that they` suggests some sort of factual basis.

> Also, the thing with replacing ?! with ?; why not just put an ! at the end of the lines where you want that?

I was lazy.

File: data/human/hails.txt [hunk 17]
```diff
@@ -1115,33 +1116,25 @@ phrase "friendly civilian"
...
 	word
-		" "
+		" while I had them on my ship"
+		" while they stayed on my ship"
+		" and wouldn't admit to it"
+		" and tried to blame it on my crew"
+		" and tried to get out of cleaning it up"
+		"" 5
+	word
+		". "
```
[the remark this answers is not in the document]
Comment (bene-dictator) [reply data/human/hails.txt:1137]: To be honest I'm tempted to remove all of the food ones.

File: data/human/hails.txt (hunk 12 above)
Comment (bene-dictator) [reply data/human/hails.txt]: Okay, using `replace` I should have fixed it.

**Review by roadrunner56 — APPROVED** [review]

File: data/human/hails.txt [hunk 18]
```diff
@@ -1986,39 +1981,69 @@ phrase "friendly civilian"
...
+		"seen how the Quarg decimate pirates with their Skylances?"
+		"seen how powerful those Quarg Skylances are?"
+	word
+		"" 3
+		" I wish the Quarg would share some of their technology with us."
+		" I'd love to get my hands on some Quarg tech."
+		" What I could do with some of that tech..."
+		" Maybe someday we'll reach their level of technology."
+		" It's scary how advanced their civilization is."
+		" There's no way humanity could stand up to the Quarg."
```
Comment (tibetiroka) [inline data/human/hails.txt:2045]: This feels less repetitive.
```suggestion
		" There's no way humanity could stand up to the them."
```

**Review by tibetiroka — CHANGES_REQUESTED** [review]

File: data/human/hails.txt (hunk 18 above)
Comment (warp-core) [reply data/human/hails.txt:2045]: "to the them"
You probably meant:
```suggestion
		" There's no way humanity could stand up to them."
```

File: data/human/hails.txt [hunk 19]
```diff
@@ -1982,24 +1969,184 @@ phrase "friendly civilian"
 		"Don't get on the bad side of the authorities. "
 		"I always try to be an honest captain. "
 		"Crime doesn't pay in the end. "
-		""
+		"You've got to be careful if you associate yourself with the wrong sorts. "
```
[the remark this answers is not in the document]
Comment (bene-dictator) [reply data/human/hails.txt:1972]: Thank goodness for `replace`. I'll fix this.

File: data/human/hails.txt [hunk 20]
```diff
@@ -1982,24 +1969,184 @@ phrase "friendly civilian"
...
 	word
 		" that "
+		", "
+	word
+		"smuggling is a sure way to find yourself behind bars"
+		"even the most powerful warship won't save you from a knife in the back"
+		"nine out of ten pirates die within their first year"
+		"Republic Intelligence has their eyes everywhere"
+		"the law will always catch up to those who try to bypass it"
+		"associating with the wrong sorts will get you dead or behind bars"
```
Comment (bene-dictator) [inline data/human/hails.txt:1991]:
```suggestion
```

File: data/human/hails.txt [hunk 21]
```diff
@@ -6098,6 +6187,29 @@ phrase "hostile pirate"
...
+phrase "hostile disabled pirate"
+	word
+		"Why isn't the ship moving?!"
+		"Why aren't the engines working?"
+		"Move, you goddamned ship!"
+		"Why aren't the guns firing?!"
+		"Come on, fire the bloody guns!"
+		"Fire! Fire! Fire the damn guns! Why aren't you firing?"
+		"The guns... they've stopped!"
+		"Weapons systems critical! What does that mean?"
```
Comment (warp-core) [inline data/human/hails.txt:6201]: Can there be less swearing here?

Comment (bene-dictator) [reply data/human/hails.txt:6201]: Ok.

**Review by tibetiroka — APPROVED** [review]

**Pull request merged**